meta.go

```diff
@@ -21,6 +22,11 @@ import (
 	"gopkg.in/urfave/cli.v1"
 )
 
+const (
+	kDefaultMetaFile  = "meta"
```

Comment:
what does the leading "k" stand for?  `golint` says:
`meta.go:26:2: don't use leading k in Go names; const kDefaultMetaFile should be defaultMetaFile`

same with `kDefaultMetaSpace` on the next line.

Reply:
Sorry… hadn't reviewed style but _had_ seen that elsewhere so was tainted… will fix.

Reply:
Done